### ISSUE TITLE
Trouble connecting/accessing information from cluster

### ISSUE THREAD

**Issue by vmarcel** [Goyoo/node-k8s-client#44]
Hello, 
I am trying to use this package to connect to Google Container Engine, which uses kubernetes. 
The issue is I am unsure why I am getting either no value or undefined when I am trying to get a simple list of the pods.
here is my config:

```javascript
var kubectl = k8s.kubectl({
  endpoint: 'https://[cluser ip]',
  namespace: 'default',
  binary: '/usr/local/bin/kubectl',
  auth: {
    username: '[user]',
    password: '[pass]'
  }
})
```
How I am calling the function:
```javascript
  kubectl.pod.list(function (err, pods) {
    console.log(pods)
  })
```

any reason why I  this is not working?

**Comment by khaled-nabil** [comment]
Have you managed to get it to work? I am running through the same issue.

**Comment by emanuelecasadio** [comment]
Yes, I got it working but now I don't use it anymore. However, I extracted the three .pem files from the `~/.azure` or `~/.kube` directory (they were encoded in base64 within a configuration file, if I can remember) and just used those files to access the cluster.
